actions
internalError: anError
	^ self handleError: anError